docs/source/guide/zne-5-theory.myst

```diff
@@ -0,0 +1,110 @@
+---
+jupytext:
+  text_representation:
+    extension: .myst
+    format_name: myst
+    format_version: 0.13
+    jupytext_version: 1.11.4
+kernelspec:
+  display_name: Python 3
+  language: python
+  name: python3
+---
+
+
+# What is the theory behind ZNE?
+
+
+Zero noise extrapolation (ZNE) {cite}`Giurgica_Tiron_2020_arXiv` is an error
+mitigation technique used to extrapolate the noiseless expectation value of an
+observable from a range of expectation values computed at different noise levels.
+In Mitiq, this process works in two steps:
+
+- **Step 1: Intentionally scale noise** by manually inserting gates in a manner
+that keeps the logic of input circuit unchanged. The number of gates inserted is
+dependent on the scale factor. Different scale factor values will calculate the
+expectation value at different noise leveles.
```

Comment:
```suggestion
- **Step 1: Intentionally scale noise**. This can be done with different methods.
*Pulse-stretching* {cite}`Temme_2017_PRL` can be used to increase the noise level
of a quantum computation. Similar results can be obtained, at a gate-level, with *unitary folding*
{cite}`Li_2017_PRX, Giurgica_Tiron_2020_arXiv`.
```
The noise can be scaled in different ways, for example pulse-stretching.